catalogd/internal/controllers/core/clustercatalog_controller_test.go

```diff
@@ -927,7 +920,7 @@ func TestPollingReconcilerUnpack(t *testing.T) {
 				},
 				Status: successfulUnpackStatus(),
 			},
-			storedCatalogData: successfulStoredCatalogData(metav1.Now()),
+			storedCatalogData: successfulStoredCatalogData(time.Now()),
```

Comment:
These `metav1.Now()` to `time.Now()` changes are there because our image handling libraries should not be coupled to kubernetes-isms. I moved conversion to `metav1.Time` into the reconciler code, iirc.